gohermes/midhermes/local/data_store.c

```diff
@@ -0,0 +1,107 @@
+#include "data_store.h"
+
+// hermes_credential_store_destroy implement hermes/mid_hermes_ll_interfaces/data_store.h interface
+hermes_status_t hermes_data_store_destroy(hermes_data_store_t **data_store){
+    return HM_SUCCESS;
```

Comment:
always success!

Reply:
yes :) because it's empty struct that allocated as field (not as pointer to struct in heap) of go struct that will be freed by go gc